src/temp/ReviewReports/SingleReview.tsx

```diff
@@ -0,0 +1,65 @@
+import { Trans, useTranslation } from 'next-i18next';
+import styles from './ReviewReports.module.scss';
+import DownloadReportIcon from '../icons/DownloadReportIcon';
+import VerifiedIcon from '@mui/icons-material/Verified';
+import { getPDFFile } from '../../utils/getImageURL';
+import parse from 'date-fns/parse';
+import format from 'date-fns/format';
+import { localeMapForDate } from '../../utils/language/getLanguageName';
+import { Review } from '@planet-sdk/common';
+
+interface Props {
+  singleReview: Review;
+}
+
+const SingleReview = ({ singleReview }: Props) => {
+  const { t } = useTranslation(['common', 'manageProjects', 'projectDetails']);
+  const displayDate = (date: string) => {
+    return format(parse(date, 'MM-yyyy', new Date()), 'LLLL yyyy', {
+      locale: localeMapForDate[localStorage.getItem('language') || 'en'],
+    });
+  };
+  return (
+    <div className={styles.singleReviewContainer}>
+      <div className={styles.reviewInfoContainer}>
+        <div className={styles.verifiedIcon}>
+          <VerifiedIcon
+            sx={{
+              color: `${'rgba(var(--review-font-color-new))'}`,
+              width: 16,
+              height: 16,
+            }}
+          />
+        </div>
```

Comment:
You could remove the `.verifiedIcon` `<div>` and just provide `align-items: center` to the `reviewInfoContainer`